target/ppc/translate/vsx-impl.c.inc

```diff
@@ -2136,6 +2136,11 @@ static int pack_flags_xvi(int acc, int sat)
     return (sat << 1) | acc;
 }
 
+static int pack_masks(int pmsk, int ymsk, int xmsk)
+{
```

Comment:
Suggested comment to add before the group of ger_get_*msk and ger_pack_masks functions:
```
/*
 * Auxiliary functions to pack/unpack masks for GER instructions.
 *
 * Packed format:
 *  Bits 0-3: xmsk
 *  Bits 4-7: ymsk
 *  Bits 8-15: pmsk
 */
```